docs/source/interfaces/isis_sans/Sum Runs.rst

```diff
@@ -0,0 +1,138 @@
+| **Run Summation List**    | This list contains the files to be summed.                                              |
++---------------------------+-----------------------------------------------------------------------------------------+
+| **Browse**                | This button is used to select one or more nexus files to be added to the summation      |
+|                           | table.                                                                                  |
++---------------------------+-----------------------------------------------------------------------------------------+
+| **Manage Directories**    | Opens the 'Manage User Directories' window allowing the user to add/remove directories  |
+|                           | from the mantid search path and to set the 'Output Folder' where the summation result   |
+|                           | will be saved.                                                                          |
++---------------------------+-----------------------------------------------------------------------------------------+
+| **Remove**                | Removes an entry from the summation table. Note, this does not delete the file itself,  |
+|                           | it just removes it from the list of files to be summed.                                 |
++---------------------------+-----------------------------------------------------------------------------------------+
+| **Remove All**            | Removes all entries from the summation table. As above, this will only remove the       |
+|                           | entries from the table, not the files themselves.                                       |
++---------------------------+-----------------------------------------------------------------------------------------+
+
+Histogram Binning
+^^^^^^^^^^^^^^^^^
+
+This panel allows the user to specify the binning parameters to be applied
+when summing event data. There are three different ways to add files containing
+event data [#no-effect-when-non-event]_.
+
+Custom Binning
+""""""""""""""
+
+If this option is chosen a line edit field becomes available which the user
+can use to set the preferred binning boundaries. The format of this input is
+identical to the format required by
+the :doc:`Rebin Algorithm </algorithms/Rebin-v1>`.
+
+
+Use Binning From Monitors
+"""""""""""""""""""""""""
+
+If this option is chosen the binning is taken from the monitors.
+
+Save As Event Data
+""""""""""""""""""
+
+If this option is chosen, the output file will contain event data.
+The output is not an event workspace but rather a group workspace, which
+contains two child event workspaces, one for the added event data and one
+for the added monitor data.
+
+With **'Overlay Event Workspaces' Disabled** the event data from the files
+is added using the event the :doc:`Plus Algorithm <../algorithms/Plus-v1>`.
```

Comment:
This link is wrong  we should be using :ref:`algm-Plus` rather than a :doc: link